modules/pipeline/spec/pipeline_cron.go

```diff
@@ -148,3 +148,28 @@ func (pc *PipelineCron) GetOrgID() uint64 {
 	orgID, _ := strconv.ParseUint(orgIDStr, 10, 64)
 	return orgID
 }
+
+func (pc *PipelineCron) GenCreatePipelineReqNormalLabels(triggerTime time.Time) map[string]string {
```

Comment:
GenCompensateCreatePipelineReqNormalLabels

it is for compenstate

Reply:
adjusted